### ISSUE TITLE
Trigger Node - [Create]> For constraint trigger only After event should displayed,  [Properties Panel/Edit Properties]> Incorrect values for Constraint trigger, Deferrable, Deferred (RM #6803)

### ISSUE THREAD

**Issue by dpage** [postgres/pgadmin4#4594]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/6803***
*Originally created by **Yogesh Mahajan** at 2021-09-24 05:13:02 UTC.*

[Create]> 
For constraint trigger only After event should displayed,  

[Properties Panel/Edit Properties]>
1.Incorrect values for Constraint trigger, Deferrable, Deferred.

2.Accessibility issues for Properties Panel > Defifnition > Trigger functions, Arguments

Sql - 
<pre>
CREATE TABLE IF NOT EXISTS public.on_duty1
(
    prison_id integer NOT NULL,
    guard_id integer NOT NULL,
    CONSTRAINT on_duty_pkey1 PRIMARY KEY (prison_id, guard_id)
)

TABLESPACE pg_default;

ALTER TABLE IF EXISTS public.on_duty1
    OWNER to postgres;

CREATE OR REPLACE FUNCTION public.check_account_update()
    RETURNS trigger
    LANGUAGE 'plpgsql'
    COST 100
    VOLATILE NOT LEAKPROOF
AS $BODY$
begin
select 1;
end;
$BODY$;

ALTER FUNCTION public.check_account_update()
    OWNER TO postgres;

CREATE CONSTRAINT TRIGGER trig
    AFTER UPDATE 
    ON public.on_duty1
    DEFERRABLE INITIALLY DEFERRED
    FOR EACH ROW
    EXECUTE FUNCTION public.check_account_update();
</pre>

**Comment by dpage** [comment]
Issue closed on Redmine.
